static/js/around-map.js

```diff
@@ -176,13 +176,12 @@ function scrollCard (poi_id) {
 // close form
 const form = document.querySelector('#poi-form')
 form.addEventListener('click', event => {
-  if(form.classList[0] === 'visible') {
+  if(event.target === form) {
```

Comment:
`event.target` is **always** equals to form.
You should just use `event.target`